typings/index.d.ts

```diff
@@ -21,6 +21,8 @@ declare module 'discord.js' {
 
   //#region Classes
 
+  type ActivityPlatform = 'xbox' | 'samsung';
```

Comment:
`type`s should go into the `Typedefs` region.

Above `ActivityType` on line 2137.